config/routes.rb

```diff
@@ -15,6 +15,8 @@ def content_id_constraint(request)
 
     post "/lookup-by-base-path", to: "lookups#by_base_path"
 
+    post "/graphql", to: "graphql#execute"
```

Comment:
Should this be in the `v2` namespace?

Reply:
I think it's fine to leave it outside that namesapce. `v2` was to indicate the second version of Publishing API's API, but this is the first iteration of the GraphQL API, which will remain separate from the other API.